alvr/xtask/src/packaging.rs

```diff
@@ -87,7 +87,7 @@ pub fn package_streamer(gpl: bool, root: Option<String>, appimage: bool, zsync:
         command::zip(&sh, &afs::streamer_build_dir()).unwrap();
 
         // the wix file broke
```

Comment:
This comment can be removed

Reply:
its such a small change, you should have pushed the changes onto the pr, instead of waiting for me.

Reply:
I will start working on my pc again from tomorrow